### PR TITLE
npm shrinkwrap

### PR DESCRIPTION
**Pull request by OliverJAsh**
Occasionally we come across weird bugs that happen because an npm module has automatically updated (because of npm version ranges in dependencies and transitive dependencies). We can avoid this by locking versions using `npm shrinkwrap` (or `npm shrinkwrap --dev`, rather—to include `devDependencies`).

> This command locks down the versions of a package's dependencies so that you can control exactly which versions of each dependency will be used when your package is installed. The package.json file is still required if you want to use npm install.
> 
> This generates npm-shrinkwrap.json, which will look something like this:
> 
> ```
> {
>   "name": "A",
>   "version": "0.1.0",
>   "dependencies": {
>     "B": {
>       "version": "0.0.1",
>       "dependencies": {
>         "C": {
>           "version": "0.0.1"
>         }
>       }
>     }
>   }
> }
> ```

Please see the documentation on `npm shrinkwrap` for more details, and instructions on how to update dependencies which are currently locked: https://docs.npmjs.com/cli/shrinkwrap

I'm aware we used to use this but it got removed. Why? Could we review again?

### CODE REVIEW

**Comment by heralder99** [comment]
NAVER - http://www.naver.com/
--------------------------------------------

heralder@naver.com 님께 보내신 메일 <[frontend] Oja shrinkwrap (#10052)> 이 다음과 같은 이유로 전송 실패했습니다.

--------------------------------------------

받는 사람이 회원님의 메일을 수신차단 하였습니다. 


--------------------------------------------

**Comment by heralder99** [comment]
NAVER - http://www.naver.com/
--------------------------------------------

heralder@naver.com 님께 보내신 메일 <Re: [frontend] npm shrinkwrap (#10052)> 이 다음과 같은 이유로 전송 실패했습니다.

--------------------------------------------

받는 사람이 회원님의 메일을 수신차단 하였습니다. 


--------------------------------------------

File: package.json [hunk 1]
```diff
@@ -74,6 +74,7 @@
     "megalog": "^0.0.2",
     "mkdirp": "0.5.0",
     "moment": "2.8.3",
+    "shoe": "0.0.11",
```
Comment (piuccio) [inline package.json]: What's this?

Comment (OliverJAsh) [inline package.json]: See the related issue in the commit https://github.com/guardian/frontend/commit/6a9ef7cabdd9823f4937629bde9003c10cc4f082

Comment (piuccio) [inline package.json]: :cry:

**Comment by piuccio** [comment]
:+1:

File: package.json (hunk 1 above)
Comment (OliverJAsh) [inline package.json]: Yup… :cry:

**Comment by desbo** [comment]
:+1:  worth trying out

**Pull request merged**